inorbit_edge/robot.py

```diff
@@ -903,6 +934,7 @@ def __init__(self, **robot_session_kw_args):
         """
         self.robot_session_kw_args = robot_session_kw_args
         self.command_callbacks = []
+        self.executable_commands_definitions = []
```

Comment:
suggest rename of this and its accessor methods to `commands_paths` or something like that